KPI-team1-frontend/src/pages/KpiPage.tsx

```diff
@@ -108,10 +119,24 @@ const data: GridRowsProp = [
 const periodicityOrder = ["daily", "weekly", "monthly", "quarterly", "yearly"];
 
 export default function KpiPage(): JSX.Element {
+  const { circleId } = useParams();
   const [modalIsOpen, setModalIsOpen] = useState(false);
   const [selectedKpi, setSelectedKpi] = useState<KpiExtended | null>(null);
   const [kpiDefinitions, setKpiDefinitions] = useState<KpiExtended[]>([]);
-  const [currentCircle, setCurrentCircle] = useState<number>(1); //TODO: replace any with the correct type and use the variable from the sidebar
+  const [circleName, setCircleName] = useState("");
+  const { circles }: OutletContext = useOutletContext();
```

Comment:
Not sure where it comes from. but if we are not logged-in, the address  page is localhost:3000/kpi
and it throws then an error 
![image](https://github.com/WomenPlusPlus/deploy-impact-23-kpi-1/assets/42074299/85843853-29f5-4b65-906f-0561301d9a08)

Reply:
Yes, as I explained in the daily meeting that we wait for favourite circle to done with this route.